osbuild/monitor.py

```diff
@@ -317,7 +317,7 @@ def __init__(self, fd: int, manifest: osbuild.Manifest):
     def begin(self, pipeline: osbuild.Pipeline):
         self._context.set_pipeline(pipeline)
         if pipeline.stages:
-            self._progress.sub_progress = Progress("stages", len(pipeline.stages))
+            self._progress.sub_progress = Progress(f"pipeline: {pipeline.name}", len(pipeline.stages))
```

Comment:
Thanks for this, would you mind updating the test, something like:
```python
diff --git a/test/mod/test_monitor.py b/test/mod/test_monitor.py
index 75fb9ac6..b159983e 100644
--- a/test/mod/test_monitor.py
+++ b/test/mod/test_monitor.py
@@ -233,6 +233,7 @@ def test_json_progress_monitor():
         logitem = json.loads(log[i])
         assert logitem["message"] == "Starting pipeline test-pipeline-first"
         assert logitem["context"]["pipeline"]["name"] == "test-pipeline-first"
+        assert logitem["progress"]["progress"]["name"] == "pipeline: test-pipeline-first"
         # empty items are omited
         assert "name" not in logitem["context"]["pipeline"]["stage"]
         i += 1
@@ -276,6 +278,7 @@ def test_json_progress_monitor():
 
         logitem = json.loads(log[i])
         assert logitem["message"] == "Starting pipeline test-pipeline-second"
+        assert logitem["progress"]["progress"]["name"] == "pipeline: test-pipeline-second"
         assert logitem["context"]["origin"] == "osbuild.monitor"
         assert logitem["context"]["pipeline"]["name"] == "test-pipeline-second"
         i += 1
```

Reply:
Thanks for implementing for me :-/ 
I'll commit in a second....

Reply:
Uh, I'm sorry, should I not to this? Reading the `:-/` make me feel you are not happy but it's hard to read through a 2d text screen :-/ If so - I'm sorry!

Reply:
all good - thanks for taking over my work and implementing the tests :-D